include/deal.II/dofs/dof_accessor.h

```diff
@@ -1588,6 +1588,9 @@ class DoFCellAccessor :  public DoFAccessor<DH::dimension,DH, level_dof_access>
    * @{
    */
 
+  // Necessary for Visual Studio for not to replace the get_fe version with one parameter with the following parameter-less version.
+  using DoFAccessor<DH::dimension, DH, level_dof_access>::get_fe;
+
```

Comment:
Right. But where does this trigger? I'm curious where MS VC tries to call the one-argument function where the other compilers are quite happy the way it is. I think that by importing the name, that we only paper over a problem that lies deeper.